examples/chart.md

```diff
@@ -133,12 +119,6 @@ Expect something like:
 > │   └── kustomization.yaml
 > ├── dev
 > │   └── kustomization.yaml
-> ├── kustomize
-> │   └── plugin
-> │       └── someteam.example.com
-> │           └── v1
-> │               └── chartinflator
-> │                  └── ChartInflator
```

Comment:
We can remove the discussion of the old inflator, but we must retain the old inflator since people actually use it.

We need to add a short explanation here of how to migrate (change the apiVersion field to` builtin`).
Users of this pluging will need time to migrate, so we need to keep both versions.
It's ok if they are both called ChartInflator, since they are in different api groups.